src/consensus/params.h

```diff
@@ -25,6 +37,10 @@ struct Params {
     int64_t nPowTargetSpacing;
     int64_t nPowTargetTimespan;
     int64_t DifficultyAdjustmentInterval() const { return nPowTargetTimespan / nPowTargetSpacing; }
+    uint64_t MaxBlockSize(uint64_t nCurrentTime) const { return blockParams->MaxBlockSize(nCurrentTime); }
```

Comment:
We may need to expose Consensus::Params in libconsensus' C API and thus it cannot have methods.
See https://github.com/jtimon/bitcoin/commit/c4cfb5e098d23e017f98aad93698350bb91960b4